src/components/config/config.ts

```diff
@@ -90,8 +91,15 @@ export async function setActiveKubeconfig(kubeconfig: string): Promise<void> {
     await addPathToConfig(KUBECONFIG_PATH_KEY, kubeconfig);
 }
 
-export function getActiveKubeconfig(): string {
-    return vscode.workspace.getConfiguration(EXTENSION_CONFIG_KEY)[KUBECONFIG_PATH_KEY];
+export function getKubeconfig(): string {
```

Comment:
>getKubeconfig [](start = 16, length = 13)

Please consider renaming to "getKubeconfigPath", so that it is clear that this is the path of the kubeconfig and not its content.

Reply:
Good point.